planning/obstacle_avoidance_planner/src/node.cpp

```diff
@@ -1499,6 +1499,7 @@ ObstacleAvoidancePlanner::alignVelocity(
   size_t prev_begin_idx = 0;
   for (size_t i = 0; i < fine_traj_points_with_vel.size(); ++i) {
     const auto truncated_points = points_utils::clipForwardPoints(path_points, prev_begin_idx, 5.0);
+    if (truncated_points.size() < 1) continue;
```

Comment:
```suggestion
    if (truncated_points.empty()) continue;
```

Reply:
@shulanbushangshu 
Thank you for testing. But I  re-think my [fix PR](https://github.com/autowarefoundation/autoware.universe/pull/1369#issuecomment-1188763184) is irrelevant to https://github.com/autowarefoundation/autoware.universe/issues/798 so can you keep confirm this PR fix the issue?